packages/authentication/docs/controller-functions.md

```diff
@@ -48,10 +48,16 @@ class LoginController{
     @inject(AuthenticationBindings.SERVICES.JWT_TOKEN) JWTtokenService: TokenService,
   ) {}
 
+  // I was about to create a local login example, while if the credentials are
+  // provided in the request body, all the authenticate logic will happen in the
+  // controller, the auth action isn't even involved.
+  // See the login endpoint in shopping example
+  // https://github.com/strongloop/loopback4-example-shopping/blob/master/src/controllers/user.controller.ts#L137
+
   // Describe the response using OpenAPI spec
-  @post('/loginOAI/local', RESPONSE_SPEC_FOR_JWT_LOGIN)
+  @post('/loginOAI/basicAuth', RESPONSE_SPEC_FOR_JWT_LOGIN)
```

Comment:
Do we want to clarify that the `login` method implementation can have two choices:

1. `@authenticate + user code`
2. plain `user code`

Reply:
Sounds good, would like to add https://github.com/strongloop/loopback-next/pull/2576#discussion_r267953354 here.